### ISSUE TITLE
NPE crash on Android 4.1, LG Optimus L9 (u2) with material-dialogs 0.8.5.7

### ISSUE THREAD

**Issue by greenaddress** [afollestad/material-dialogs#1026]
Have you seen this problem before?

Not sure how to reproduce but what I know thus far is:

Android 4.1, LG Optimus L9 (u2) with material-dialogs 0.8.5.7

```
java.lang.NullPointerException
	at android.support.v4.graphics.drawable.DrawableWrapperDonut.setCompatTintList(DrawableWrapperDonut.java:272)
	at android.support.v4.graphics.drawable.DrawableCompatBase.setTintList(DrawableCompatBase.java:42)
	at android.support.v4.graphics.drawable.DrawableCompat$BaseDrawableImpl.setTintList(DrawableCompat.java:94)
	at android.support.v4.graphics.drawable.DrawableCompat.setTintList(DrawableCompat.java:390)
	at com.afollestad.materialdialogs.internal.MDTintHelper.setTint(MDTintHelper.java:45)
	at com.afollestad.materialdialogs.DefaultAdapter.getView(DefaultAdapter.java:67)
	at android.widget.AbsListView.obtainView(AbsListView.java:2306)
	at android.widget.ListView.measureHeightOfChildren(ListView.java:1244)
	at android.widget.ListView.onMeasure(ListView.java:1156)
	at android.view.View.measure(View.java:15293)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:310)
	at android.view.View.measure(View.java:15293)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1418)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:709)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:602)
	at android.view.View.measure(View.java:15293)
	at com.afollestad.materialdialogs.internal.MDRootLayout.onMeasure(MDRootLayout.java:199)
	at android.view.View.measure(View.java:15293)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:310)
	at android.view.View.measure(View.java:15293)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1418)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:709)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:602)
	at android.view.View.measure(View.java:15293)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:310)
	at com.android.internal.policy.impl.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2425)
	at android.view.View.measure(View.java:15293)
	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:1876)
	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1087)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1301)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1010)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:4255)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:725)
	at android.view.Choreographer.doCallbacks(Choreographer.java:555)
	at android.view.Choreographer.doFrame(Choreographer.java:525)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:711)
	at android.os.Handler.handleCallback(Handler.java:615)
	at android.os.Handler.dispatchMessage(Handler.java:92)
	at android.os.Looper.loop(Looper.java:137)
	at android.app.ActivityThread.main(ActivityThread.java:4849)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:511)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:795)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:562)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by AntonyGolovin** [comment]
Just got same crash.

**Comment by colinrtwhite** [comment]
+1 I have the same problem (no issue on 0.8.5.6). Tested on a Nexus S 4.1 (emulator).

Though, I think this is a duplicate of #1006, which has been marked as fixed. Best to just stick on 0.8.5.6 and wait for the next release.

**Comment by yorkwang** [comment]
Also meet this problem on 4.4.2

**Comment by lukef** [comment]
This seems fixed in 0.8.5.8 for me
